accepted/PSR-7-http-message-meta.md

```diff
@@ -252,26 +252,27 @@ request or reset request state for each message the client sends:
 
 ~~~php
 $uri = new Uri('http://api.example.com');
-$baseRequest = new Request($uri, null, [
-    'Authorization' => 'Bearer ' . $token,
-    'Accept'        => 'application/json',
-]);;
+$baseRequest = new Request(
+    $uri, 
+    null, 
+    [
+        'Authorization' => 'Bearer ' . $token,
+        'Accept'        => 'application/json',
```

Comment:
I don't think that using that rule here is relevant.

Reply:
replace to ...?

Reply:
Remove empty extra spaces

Reply:
fixed